webpack.config.common.js

```diff
@@ -0,0 +1,308 @@
+const webpack = require('webpack');
+const PACKAGE = require('./package.json');
+const CopyPlugin = require('copy-webpack-plugin');
+const MergeIntoSingleFilePlugin = require('webpack-merge-and-include-globally');
+const {merge} = require("webpack-merge");
+const singleSpaDefaults = require("webpack-config-single-spa");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+
+// Pass this function as a transform argument to CopyPlugin elements to replace [AIV]{version}[/AIV]
+// with the current workbench version number. This is not related to the WebpackAutoInject plugin
+// (it will replace only in bundled files, not in CopyPlugin) but we use the same tag for consistency.
+function replaceVersion(content) {
+    return content
+        .toString()
+        .replace(/\[AIV]{version}\[\/AIV]/g, PACKAGE.version);
+}
+
+const host = 'localhost';
+const portHere = 9000;
+const portThere = 7200;
+
+module.exports = (webpackConfigEnv, argv) => {
+    const orgName = "ontotext";
+    const defaultConfig = singleSpaDefaults({
+        orgName,
+        projectName: "root-config",
+        webpackConfigEnv,
+        argv,
+        disableHtmlGeneration: true,
+    });
+
+    defaultConfig.module.rules = [];
+
+    return merge(defaultConfig, {
+        entry: {
+            main: './packages/root-config/src/ontotext-root-config.js',
+            legacyWorkbench: './packages/legacy-workbench/src/index.js'
+        },
+        output: {
+            filename: '[name].js',
+            chunkFilename: '[name].bundle.js',
+            path: path.resolve(__dirname, 'dist'),
+            libraryTarget: "system",
+            publicPath: '/'
+        },
+        resolve: {
+            modules: [
+                './packages/legacy-workbench/src/js/',
+                './packages/legacy-workbench/node_modules',
+                './packages/root-config/src/js/',
+                './packages/root-config/node_modules'
+            ],
+            extensions: ['.js']
+        },
+        externals: ["@ontotext/workbench-api", "@ontotext/components"],
```

Comment:
`@ontotext/components` check this out as well! We don't have such module

Reply:
Done